lib/main.dart

```diff
@@ -10,12 +12,15 @@ class MyApp extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    return MaterialApp(
-      title: 'Playtify',
-      theme: ThemeData(
-        primarySwatch: Colors.blue,
+    return ChangeNotifierProvider<RecentlyPlayedLogic>(
+      create: (_) => RecentlyPlayedLogic(),
+      child: MaterialApp(
+        title: 'Playtify',
+        theme: ThemeData(
```

Comment:
switch to material-3
Add ..
> useMaterial3: true

refer [Material-3-Flutter](https://docs.flutter.dev/ui/design/material)